pipelines/matrix/src/matrix/pipelines/disease_list/nodes.py

```diff
@@ -0,0 +1,818 @@
+"""Disease list pipeline nodes for MATRIX.
+
+This pipeline processes the MONDO disease ontology to create the MATRIX disease list,
+which defines which diseases are included in the drug repurposing platform.
+
+The workflow is modeled after the matrix-disease-list repository:
+https://github.com/everycure-org/matrix-disease-list
+"""
+
+import logging
+import re
+import tempfile
+from pathlib import Path
+from typing import Any, Dict, List, Optional
+
+import pandas as pd
+
+from matrix.utils.system import run_subprocess
+
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s")
+logger = logging.getLogger(__name__)
+from oaklib.datamodels.vocabulary import IS_A
+
+
+def extract_mondo_labels(mondo_owl: str) -> pd.DataFrame:
+    """Extract disease labels from MONDO ontology using ROBOT.
+
+    Args:
+        mondo_owl: MONDO ontology content in OWL format
+
+    Returns:
+        DataFrame with disease IDs and labels
+    """
+    logger.info("Extracting MONDO labels from ontology")
+
+    with tempfile.TemporaryDirectory() as tmpdir:
```

Comment:
Should be handled by the catalog

Reply:
So what is the preferred way to invoke a CLI tool if I must? This IO is only necessary because the tool need files.